kratos/tests/test_flux_based_redistance.py

```diff
@@ -0,0 +1,200 @@
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics
+from KratosMultiphysics import python_linear_solver_factory as linear_solver_factory
+import KratosMultiphysics.vtk_output_process as vtk_output_process
+import math
+import os
+
+def GetFilePath(fileName):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
+
+class TestRedistance(KratosUnittest.TestCase):
+
+    def _ExpectedDistance(self,x,y,z):
+        d = x
+        if( d > 0.2):
+            d = 0.2
+        if( d < -0.2):
+            d = -0.2
+        return x
+        #return -(math.sqrt(x**2+y**2+z**2) - 0.4)
```

Comment:
Is this correct?

Reply:
removed commented code, it's just a plane